draft-ietf-quic-applicability.md

```diff
@@ -664,8 +680,8 @@ will persist only for as long as any Version Negotiation packets take to be
 transmitted and responded to.  So the third stage can follow after a relatively
 short delay.
 
-The third stage completes the process by enabling validation of the negotiation
-version as though the new version were disabled.
+The third stage completes the process by enabling validation of the negotiated
+version with the assumption that the new version is fully available.
```

Comment:
Actually I'm not sure here. If I remember correctly this text was provide by someone else and it not entirely clear to me anymore what this stage is suppose to achieve....

Reply:
This was my text originally.  But I got it backwards.  You can't complete an upgrade that adds a new version and end with the assumption that the new version is disabled.  I probably meant "enabled".

Reply:
Ah okay. However, what's actually meant by validation in this phase?

Reply:
Validation being authentication of the values.  Maybe I can use that instead.